README.md

```diff
@@ -1,2 +1,46 @@
-# labirynt
-Prosta gra w labirynt tworzona dla demonstracji używania gita w grupie.
\ No newline at end of file
+# Labirynt
+> Prosta gra.
+
+## Spis treści
+* [Informacje ogólne](#ogólne-informacje)
+* [Autorzy](#autorzy)
+* [Technologie](#technologie)
+* [Setup](#setup)
+* [Rozwój](#rozwój)
+* [Status](#status)
+* [Kontakt](#kontakt)
+
+## Ogólne informacje
+Ten projekt jest demonstracją używania git'a w grupie.
+
+## Autorzy
+Labirynt tworzy 5 osób:
+* Marcin Zepp
+* Igor Wysocki
+* Michał Hoffmann
+* Dawid Siekański
+* Igor Białas
+
+## Technologie
+* C ++ - standard c++98.
+
+## Setup
+Pobierz grę z https://github.com/Nircek/labirynt/releases/tag/v1.0
+i wpisz:
+
+* w Linux Debian:
```

Comment:
na Debianie?, ja bym napisał

Reply:
Przez google tłumacz to przepuściłem i nie zauważyłęm

Reply:
Naprawione w 0f453929655840bedbdc690f8932114748346ed1.